### PR TITLE
MGR updates

### PR DESCRIPTION
**Pull request by oseikuffuor1**
This PR includes some new updates to the MGR solver regarding the user input interface.

### CODE REVIEW

**Comment by oseikuffuor1** [comment]
PR passes both GPU and tux tests.

**Review by rfalgout — APPROVED** [review]
Looks good to me!  -Rob

**Review by ulrikeyang — APPROVED** [review]
Looks fine.

File: src/parcsr_ls/par_mgr.c [hunk 1]
```diff
@@ -5141,84 +5194,85 @@ hypre_MGRBuildAff( MPI_Comm comm, HYPRE_Int local_num_variables, HYPRE_Int num_f
...
 }
 
 /* Print global smoother convergence factor */
 /*
 HYPRE_Int
 hypre_MGRPrintGsmoothConvergenceFactor( void *mgr_vdata, HYPRE_Int print_flag)
 {
-   hypre_ParMGRData  *mgr_data = (hypre_ParMGRData*) mgr_vdata;
-   mgr_data->print_gsmooth_convergence_factor = print_flag;
+<<<<<<< HEAD
```
Comment (liruipeng) [inline src/parcsr_ls/par_mgr.c]: What is this? From an unfinished merge?

File: src/parcsr_ls/par_mgr.c [hunk 2]
```diff
@@ -1074,158 +1138,157 @@ hypre_MGRBuildP( hypre_ParCSRMatrix   *A,
...
-   for (i = 0; i < n_fine; i++)
-   {
-      for (jj = A_diag_i[i]; jj < A_diag_i[i+1]; jj++)
+  /*-----------------------------------------------------------------------
+  *  Loop over fine grid points.
+  *-----------------------------------------------------------------------*/
+  a_diag = hypre_CTAlloc(HYPRE_Real,  n_fine, HYPRE_MEMORY_HOST);
+  for (i = 0; i < n_fine; i++)
+  {
+    if (CF_marker[i] < 0)
```
Comment (liruipeng) [inline src/parcsr_ls/par_mgr.c:1149]: plz add curly brackets after if-block

**Review by liruipeng — APPROVED** [review]

File: src/parcsr_ls/par_mgr.c (hunk 1 above)
Comment (mquanbui) [reply src/parcsr_ls/par_mgr.c]: Thanks for spotting this. It is inside a block that is commented out for removal. I'm gonna remove this section.

File: src/parcsr_ls/par_mgr.c (hunk 2 above)
Comment (mquanbui) [reply src/parcsr_ls/par_mgr.c:1149]: The code is correct. Added the brackets for good practice/readability.

**Review by mquanbui — APPROVED** [review]

**Pull request merged**